includes/class-wc-stripe-helper.php

```diff
@@ -373,7 +375,20 @@ public static function get_webhook_url() {
 	public static function get_order_by_source_id( $source_id ) {
 		global $wpdb;
 
-		$order_id = $wpdb->get_var( $wpdb->prepare( "SELECT DISTINCT ID FROM $wpdb->posts as posts LEFT JOIN $wpdb->postmeta as meta ON posts.ID = meta.post_id WHERE meta.meta_value = %s AND meta.meta_key = %s", $source_id, '_stripe_source_id' ) );
+		if ( class_exists( 'OrderUtil' ) && OrderUtil::custom_orders_table_usage_is_enabled() ) {
```

Comment:
The `class_exists()` call needs to use the fully qualified class name otherwise this won't work.

```suggestion
		if ( class_exists( 'Automattic\WooCommerce\Utilities\OrderUtil' ) && OrderUtil::custom_orders_table_usage_is_enabled() ) {
```